src/DataTable/ActionDisplay.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useContext } from 'react';
+import BulkActions from './BulkActions';
+import TableActions from './TableActions';
+import DataTableContext from './DataTableContext';
+
+// handles layout for filters, status, and bulk actions
+const ActionDisplay = () => {
+  const { bulkActions, tableActions, selectedFlatRows } = useContext(DataTableContext);
+  const noBulkActions = bulkActions?.length < 1;
+  const noTableActions = tableActions?.length < 1;
+  const noActionsAvailable = noBulkActions && noTableActions;
+  const noRowsSelected = selectedFlatRows?.length < 1;
+
+  // return null if there are no actions or if there are only bulk actions and no selected rows
+  if (noActionsAvailable || (noTableActions && noRowsSelected)) {
+    return null;
+  }
+  if (!noRowsSelected) {
```

Comment:
nit: got slightly confused by the double negative here. might consider renaming this variable to `hasRowsSelected` and checking for a falsey value in the above conditional?

Reply:
Yeah, I debated doing that, will change.